src/Administration/Resources/app/administration/src/app/component/media/sw-media-upload-v2/index.js

```diff
@@ -450,24 +450,38 @@ Component.register('sw-media-upload-v2', {
                 return true;
             }
 
-            const fileTypes = this.fileAccept.split(',');
+            const fileTypes = this.fileAccept.split(',').map(s => s.trim());
 
-            fileTypes.forEach(fileType => {
-                const fileAcceptType = fileType.split('/');
-                const currentFileType = file?.type?.split('/') || file?.mimeType?.split('/');
+            // eslint-disable-next-line no-restricted-syntax
+            for (const fileType of fileTypes) {
```

Comment:
@fk-bitsandlikes Why do you want to use `for...` instead of  `fileTypes.forEach`. I don't get the idea. Furthermore, this part in the `trunk` has already changed. Could you possibly rebase your branch once please?

Reply:
I have used `for...` instead of `forEach`, because `forEach` has officially not break, see
> There is no way to stop or break a forEach() loop other than by throwing an exception. If you need such behavior, the forEach() method is the wrong tool.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach?retiredLocale=de#description

Therefore, in the current code in the `trunk`, `forEach` has been changed to `some`.

I have rebased the code fragment.